target/linux/pistachio/base-files/lib/netifd/proto/lowpan.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/sh
```

Comment:
It would be better to put this into an extra package for example into the iwpan package, this will probably also work on other devices and should be used there.